public/asset/js/Hammer.js

```diff
@@ -16,3 +16,25 @@ window.Hammer = {
 }
 
 window.ProspectFlow = window.Hammer;
+
+document.addEventListener("DOMContentLoaded", function() {
+    var delayMinutes = 5;
+    var lastFetchNotificationTime = localStorage.getItem('ProspectFlowLastFetchNotificationTime');
+    
+    if(lastFetchNotificationTime === null){
+        lastFetchNotificationTime = Date.now();
+        localStorage.setItem('ProspectFlowLastFetchNotificationTime', lastFetchNotificationTime);        
+        
+        ProspectFlow.Notification.getLatest();
+    }
+
+    var timeElapsed = Date.now() - lastFetchNotificationTime;
+    var minutesElapsed = timeElapsed / 60000;
```

Comment:
Use let inside a function instead of var for local scope var is for global DOM scope